screens/ChatScreen.js

```diff
@@ -110,62 +155,65 @@ class ChatScreen extends React.Component {
         } = this.state;
         return (
             <View style={styles.container}>
+                <Header>
+                    <Left>
+                        <Button transparent>
+                            <Icon
+                                name="md-arrow-back"
+                                onPress={() =>
+                                    navigation.pop()
+                                }
+                            />
+                        </Button>
+                    </Left>
+                    <Body>
+                        <Title>
+                            {navigation.getParam('chatName')}
+                        </Title>
+                    </Body>
+                    <Right>
+                        <Button
+                            transparent
+                            onPress={() =>
+                                this.props.navigation.navigate('AddUserToChat', {
+                                    chatId: currentChat.id
+                                })
+                            }>
+                            <Icon name="person-add" />
+                        </Button>
+                        <Button
+                            transparent
+                            onPress={this.confirmLeavingChat}>
+                            <MaterialIcon
+                                name="logout"
+                                type="material-community"
+                                color="red"
+                                iconStyle={styles.materialIcon}
+                            />
+                        </Button>
+                    </Right>
+                </Header>
                 <ScrollView
                     style={styles.chatContainer}
-                    contentContainerStyle={styles.contentContainer}>
+                    contentContainerStyle={styles.contentContainer}
+                    ref={ref => this.scrollView = ref}
+                    onContentSizeChange={() => {
+                        this.scrollView.scrollToEnd({ animated: true });
+                    }}>
```

Comment:
this doesn't actually work exactly right. i didn't want to spend a lot of time investigating this since it seems like a pretty prevalent problem that people have had to hack to get to work correctly based on whatever container the scrollview is in. so just a heads up that we'll want to revisit this later (added to google doc) and when another user posts or you post a message, you're going to need to swipe down to see it.